src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let rent = 0;
+
+  rent = days * 40;
+
+  if (days >= 7) {
+    rent -= 50;
+
+    return rent;
+  }
+
+  if (days >= 3) {
+    rent -= 20;
+
+    return rent;
+  }
+
+  return rent;
 }
 
+calculateRentalCost(5);
```

Comment:
why do you need to call this function? 🤔

Reply:
Forgot to remove(